### ISSUE TITLE
issue with github workflow

### ISSUE THREAD

**Issue by uriah-we** [frencojobs/envify#26]
I dont know if , this is the right place to post this issue,

But before i was trying to use envify, 

i added a run command of `flutter pub run build_runner build` 

i know it wont work since i dont have .env file on the repo.

so i try the method used in github workflow to declare the env 

but also it didnt work, i believe the env github workflow is using is the ENV 

on linux instance or whatever it may be on windows , but linux and mac is the same

if we created a ENV VAR,

so basically the issue is asking for a .env file that doesnt exists.

i have 2 ways to go about it, one is running a shell script and use the ENV var declared with github.

and pipe it to .env and .env.dev ...

2nd one is to simply remove from .gitignore env.g.dart 

I hope you guys have more better solution in my mind 

please if you have please share thanks

**Comment by MuTe33** [comment]
Yea, you can encode your `.env` file as `base64` and before build process pipe it in decoded using `echo "$ENV_FILE" | base64 --decode > .env`

`ENV_FILE` is the secret/env variable define by your CI
